### PR TITLE
Document running a custom MAO and actuator.

### PR DESCRIPTION
**Pull request by russellb**

### CODE REVIEW

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/495/

File: docs/custom-mao-and-capbm.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+```sh
+git clone https://github.com/openshift/machine-api-operator
+cd machine-api-operator
+# Make any necessary changes
+make machine-api-operator
+```
+
+If you have trouble building the MAO with the above command, you can try
+running podman manually.  Fix the paths to reflefct your environment, first.
+
```
Comment (hardys) [inline docs/custom-mao-and-capbm.md]: typo reflect

File: docs/custom-mao-and-capbm.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+```sh
+sudo podman run --rm -v "/home/rbryant/go/src/github.com/openshift/machine-api-operator":/go/src/github.com/openshift/machine-api-operator:Z -w /go/src/github.com/openshift/machine-api-operator golang:1.10 ./hack/go-build.sh machine-api-operator
```
Comment (hardys) [inline docs/custom-mao-and-capbm.md]: If we use $USER here it may be nicer for folks if they cut/paste from the docs?

**Comment by hardys** [comment]
Looks great, thanks for sharing, couple of minor nits noted

File: docs/custom-mao-and-capbm.md (hunk 1 above)
Comment (russellb) [reply docs/custom-mao-and-capbm.md]: thanks, fixed

File: docs/custom-mao-and-capbm.md (hunk 2 above)
Comment (russellb) [reply docs/custom-mao-and-capbm.md]: good suggestions, thanks, fixed

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/511/

**Pull request merged**